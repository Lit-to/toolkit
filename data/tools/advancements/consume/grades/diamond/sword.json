{
    "criteria": {
        "use_item": {
            "trigger": "minecraft:consume_item",
            "conditions": {
                "item": {
                    "items": [
                        "minecraft:diamond_sword"
                    ]
                }
            }
        }
    },
    "rewards": {
        "function": "tools:from_adv/grades/diamond/sword"
    }
}