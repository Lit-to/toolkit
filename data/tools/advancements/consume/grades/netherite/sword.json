{
    "criteria": {
        "use_item": {
            "trigger": "minecraft:consume_item",
            "conditions": {
                "item": {
                    "items": [
                        "minecraft:netherite_sword"
                    ]
                }
            }
        }
    },
    "rewards": {
        "function": "tools:from_adv/grades/netherite/sword"
    }
}