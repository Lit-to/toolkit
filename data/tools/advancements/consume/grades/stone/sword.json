{
    "criteria": {
        "use_item": {
            "trigger": "minecraft:consume_item",
            "conditions": {
                "item": {
                    "items": [
                        "minecraft:stone_axe"
                    ]
                }
            }
        }
    },
    "rewards": {
        "function": "tools:consume"
    }
}