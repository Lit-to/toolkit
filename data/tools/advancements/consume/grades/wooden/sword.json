{
    "criteria": {
        "use_item": {
            "trigger": "minecraft:consume_item",
            "conditions": {
                "item": {
                    "items": [
                        "minecraft:wooden_sword"
                    ]
                }
            }
        }
    },
    "rewards": {
        "function": "tools:from_adv/grades/wooden/sword"
    }
}