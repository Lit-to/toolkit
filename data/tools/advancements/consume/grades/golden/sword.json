{
    "criteria": {
        "use_item": {
            "trigger": "minecraft:consume_item",
            "conditions": {
                "item": {
                    "items": [
                        "minecraft:golden_axe"
                    ]
                }
            }
        }
    },
    "rewards": {
        "function": "tools:consume"
    }
}